### PR TITLE
HDDS-2765. security/SecureOzone.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2765

## How was this patch tested?

hugo server -D

### CODE REVIEW

File: hadoop-hdds/docs/content/security/SecureOzone.zh.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+Ozone 使用 token 的方法来防止 Kerberos 服务器负载过重，当每秒处理上千个请求时，Kerberos 可能无法很好地工作。所以，每次当用户完成一次认证之后，Ozone 会向用户颁发代理 token 和块 token，应用程序可以使用这些 token 来对集群进行特定的操作，就像它们持有 kerberos 凭据一样，Ozone 支持以下类型的 token。
+
+### 代理 Token ###
+代理 token 允许应用模拟用户的 kerberos 凭据，它基于 kerberos 的身份认证，由 OM 颁发，当集群启用安全机制时，代理 token 功能默认启用。
+
+### 块 Token ###
+
+用户通过块 token 来读写一个块，它的作用是让数据节点知道用户是否有对块进行读和修改的权限。
+
+### S3 Token ###
```
Comment (cxorm) [inline hadoop-hdds/docs/content/security/SecureOzone.zh.md:48]: In [doc of 0.5.0-beta](https://hadoop.apache.org/ozone/docs/0.5.0-beta/security/secureozone.html), the S3 Token has been changed to S3AuthInfo, could we make it the same here?

**Comment by iamabug** [comment]
> Thanks @iamabug for the translation.
> Overall LGTM, a minor title could be consistent with the latest release.

Sorry for taking so long, a new commit has been made

**Comment by adoroszlai** [comment]
@iamabug Thank you for working on this translation.  Latest CI run (for 55f1de2) is green, so I think it can be merged once @cxorm approves latest changes, no need to retest.

**Comment by cxorm** [comment]
Thanks @iamabug for the work.
And thanks @adoroszlai for triggering CI checks.
I have committed it.

**Pull request merged**